added docs from archive

maxant opened the pull request:
Hi, as requested, here is a PR for the docs. I've copied pretty much everything I could find from the archive and migrated it over to asciidoc.  You can generate the HTML out of it using `mvn generate-resouces`. Hope it meets your expectations!

Cheers,
Ant

lorban commented on btm-docs/pom.xml:
```diff
@@ -0,0 +1,39 @@
+            <plugin>
+                <groupId>org.asciidoctor</groupId>
+                <artifactId>asciidoctor-maven-plugin</artifactId>
+                <version>1.5.2.1</version> 
```
This version should be specified in the parent pom's pluginManagement section.

maxant replied:
done

lorban commented on btm-docs/pom.xml:
```diff
@@ -0,0 +1,39 @@
+                </executions>
+                <configuration>
+                    <sourceDirectory>src/main/doc</sourceDirectory>
+                    <outputDirectory>target/docs</outputDirectory>
```
Both folders should be prefixed with ${basedir}/ for peace of mind.

maxant replied:
done

lorban commented:
Why did you name the files *.txt rather than *.adoc?

lorban commented:
Don't you think it would be a good idea to add this btm-doc module to the parent pom's <modules> section?

maxant commented:
I've addressed the issues. The parent POM already referenced the new module. Files have been renamed and moved to default folders.  Pushed and this pull request now reflects the new changes. Let me know if anything else needs doing.
